samples/usb/main.c

```diff
@@ -0,0 +1,37 @@
+#include <hal/debug.h>
+#include <hal/video.h>
+#include <windows.h>
+#include <usbh_lib.h>
+
+/*
+ * This demo will initialise the usb stack, detect events on the usb ports
+ * and print the device PID and VID.
+*/
+void device_connection_callback(UDEV_T *udev, int status)
+{
+    debugPrint("Device connected on port %u (PID: %04x VID: %04x)\n", udev->port_num,

```

Comment:
What do these ports look like? If they are USB ports and not gamepad ports, then that's relevant for the message, in my opinion.

Reply:
They are the number of the USB hub port. I have updated the wording here.